sedaro/pyproject.toml

```
@@ -9,7 +9,7 @@ description = "A python client to interact with the Sedaro API."
 readme = "README.md"
 requires-python = ">=3.8"
 classifiers = [ "Programming Language :: Python :: 3.8", "License :: OSI Approved :: GNU General Public License v3 (GPLv3)", "Operating System :: OS Independent", "Topic :: Software Development",]
-dependencies = [ "certifi >= 14.5.14", "flatdict ~= 4.0.1", "frozendict ~= 2.3.4", "python-dateutil ~= 2.7", "setuptools >= 21.0.0", "typing_extensions ~= 4.3.0", "urllib3 ~= 1.26.7", "pydash >= 5.1.1", "scipy >= 1.10.1", "numpy",]
+dependencies = [ "certifi >= 14.5.14", "flatdict ~= 4.0.1", "frozendict ~= 2.3.4", "orjson ~= 3.9.0", "python-dateutil ~= 2.7", "requests ~= 2.31.0", "setuptools >= 21.0.0", "typing_extensions ~= 4.3.0", "urllib3 ~= 1.26.7", "pydash >= 5.1.1", "scipy >= 1.10.1", "numpy",]
```

`requests` is a pretty common dependency so I don't want to be too strict on the version.  Especially given that our use isn't exotic.  Lets remove the version constraint.

```suggestion
dependencies = [ "certifi >= 14.5.14", "flatdict ~= 4.0.1", "frozendict ~= 2.3.4", "orjson ~= 3.9.0", "python-dateutil ~= 2.7", "requests", "setuptools >= 21.0.0", "typing_extensions ~= 4.3.0", "urllib3 ~= 1.26.7", "pydash >= 5.1.1", "scipy >= 1.10.1", "numpy",]
```